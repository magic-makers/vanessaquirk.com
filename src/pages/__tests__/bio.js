import React from 'react';
import renderer from 'react-test-renderer';
import { StaticQuery } from 'gatsby';
import BioPage from '../Bio';

// Needed because StaticQuery is used in Layout component
beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: `Default Starter`,
        },
      },
    })
  );
});

describe('Bio', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BioPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
