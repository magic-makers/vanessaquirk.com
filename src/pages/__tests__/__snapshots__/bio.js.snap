// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders correctly 1`] = `
<div
  className="main-body is-preload"
>
  <div
    id="wrapper"
  >
    <header
      id="header"
    >
      <div
        className="inner"
      >
        <a
          className="logo"
          href="/"
        >
          <span
            className="symbol"
          >
            <img
              alt=""
              src="test-file-stub"
            />
          </span>
          <span
            className="title"
          >
            VanessaQuirk.com
          </span>
        </a>
        <nav>
          <ul>
            <li>
              <a
                href="#menu"
                onClick={[Function]}
              >
                Menu
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <div
      className=" "
    >
      <nav
        id="menu"
      >
        <div
          className="inner"
        >
          <h2>
            Menu
          </h2>
          <ul>
            <li>
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                href="/bio"
              >
                Bio
              </a>
            </li>
            <li>
              <a
                href="/articles"
              >
                Articles
              </a>
            </li>
          </ul>
        </div>
        <a
          className="close"
          href="#menu"
          onClick={[Function]}
        >
          Close
        </a>
      </nav>
    </div>
    <div
      id="main"
    >
      <div
        className="inner"
      >
        <h1>
          Bio
        </h1>
        <p>
          <span
            className="image left"
          >
            <img
              alt=""
              src="test-file-stub"
            />
          </span>
          Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent.
        </p>
        <p>
          <span
            className="image right"
          >
            <img
              alt=""
              src="images/pic15.jpg"
            />
          </span>
          Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent.
        </p>
      </div>
    </div>
    <footer
      id="footer"
    >
      <div
        className="inner"
      >
        <section>
          <h2>
            Get in touch
          </h2>
          <form
            action="#"
            method="post"
          >
            <div
              className="fields"
            >
              <div
                className="field half"
              >
                <input
                  id="name"
                  name="name"
                  placeholder="Name"
                  type="text"
                />
              </div>
              <div
                className="field half"
              >
                <input
                  id="email"
                  name="email"
                  placeholder="Email"
                  type="email"
                />
              </div>
              <div
                className="field"
              >
                <textarea
                  id="message"
                  name="message"
                  placeholder="Message"
                />
              </div>
            </div>
            <ul
              className="actions"
            >
              <li>
                <input
                  className="primary"
                  type="submit"
                  value="Send"
                />
              </li>
            </ul>
          </form>
        </section>
        <section>
          <h2>
            Follow
          </h2>
          <ul
            className="icons"
          >
            <li>
              <a
                className="icon fa-twitter"
                href="https://twitter.com/vmquirk"
              >
                <span
                  className="label"
                >
                  Twitter
                </span>
              </a>
            </li>
            <li>
              <a
                className="icon fa-facebook"
                href="https://www.facebook.com/vmquirk"
              >
                <span
                  className="label"
                >
                  Facebook
                </span>
              </a>
            </li>
          </ul>
        </section>
        <ul
          className="copyright"
        >
          <li>
            Â© Magic Makers. All rights reserved
          </li>
          <li>
            Design: 
            <a
              href="http://html5up.net"
            >
              HTML5 UP
            </a>
          </li>
        </ul>
      </div>
    </footer>
  </div>
</div>
`;
